From ae81e8494fcebeb5f149e21d8d958784eb59fe32 Mon Sep 17 00:00:00 2001
From: David Pacheco <dap@joyent.com>
Date: Wed, 14 Sep 2016 10:52:09 -0700
Subject: [PATCH] joyent/node-manta#271 document move to cr.joyent.us Reviewed
 by: Jordan Hendricks <jordan.hendricks@joyent.com> Approved by: Jordan
 Hendricks <jordan.hendricks@joyent.com>

---
 CONTRIBUTING.md | 9 +++++++++
 README.md       | 7 +++++--
 2 files changed, 14 insertions(+), 2 deletions(-)
 create mode 100644 CONTRIBUTING.md

diff --git a/CONTRIBUTING.md b/CONTRIBUTING.md
new file mode 100644
index 0000000..dad934b
--- /dev/null
+++ b/CONTRIBUTING.md
@@ -0,0 +1,9 @@
+# Contributing
+
+This repository is part of the Joyent Manta project.  See the [contribution
+guidelines for the Manta
+project](https://github.com/joyent/manta/blob/master/CONTRIBUTING.md).
+
+In addition to the guidelines described there, user-facing changes should
+include an update to CHANGES.md (to list the change) and package.json (to bump
+this package's version appropriately).
diff --git a/README.md b/README.md
index 1d30d69..ac8b2ed 100644
--- a/README.md
+++ b/README.md
@@ -1,7 +1,10 @@
 # Manta client tools
 
-[manta](http://apidocs.joyent.com/manta/nodesdk.html) is a Node.js SDK for interacting
-with Joyent's Manta system.
+[manta](http://apidocs.joyent.com/manta/nodesdk.html) is a Node.js SDK for
+interacting with Joyent's Manta system.
+
+This repository is part of the Joyent [Manta](http://github.com/joyent/manta)
+project.  See [CONTRIBUTING.md](CONTRIBUTING.md) for contribution guidelines.
 
 # Installation
 
-- 
2.21.0

